modules/helm-initializer/main.tf

```diff
@@ -26,7 +26,18 @@ data "template_file" "tiller_rbac" {
   }
 }
 
+data "external" "tiller_status" {
+  program = [
+    "bash",
+    "-c",
+    "REPLICAS=$$(kubectl get deploy tiller-deploy -n ${var.tiller_namespace} -o jsonpath='{.status.readyReplicas}'); jq -n --arg replicas \"$$REPLICAS\" '{readyReplicas:$$replicas}'"]
```

Comment:
Should these commands be joined with `&&` instead of `;`? What happens to the trigger if the kubectl command fails, e.g. due to something dumb like a connection timeout?